src/Clr2Ts.Transpiler/Transpilation/Templating/TypeScriptCodeTemplates/ClassPropertyDefinition.txt

```diff
@@ -1,2 +1,2 @@
 ﻿
-[Documentation][Decorators]public [PropertyName]: [PropertyType][Assignment];
\ No newline at end of file
+[Documentation][Decorators]public [PropertyConstrains][PropertyName]: [PropertyType][Assignment];
```

Comment:
Rename Placeholder to *PropertyModifiers*, terminology from here:
https://www.typescriptlang.org/docs/handbook/2/objects.html#property-modifiers